### PR TITLE
Fixes chat message bubble styles

### PR DESCRIPTION
**Pull request by mike-lvov**
fixes https://github.com/sparkletown/internal-sparkle-issues/issues/419

### CODE REVIEW

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/atoms/ChatMessage/ChatMessage.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ $my-text-border-radius: 28px 28px 4px 28px;
 .chat-message {
   align-self: flex-start;
   display: flex;
+  align-items: flex-start;
```
Comment (0xdevalias) [inline src/components/atoms/ChatMessage/ChatMessage.scss:9]: @mike-lvov How does adding `align-items` here differ from `align-self`?

- https://www.w3schools.com/cssref/css3_pr_align-self.asp
  - > The align-self property specifies the alignment for the selected item inside the flexible container.
  - > Note: The align-self property overrides the flexible container's align-items property.
  
Is `align-self` still needed here?

File: src/components/atoms/ChatMessage/ChatMessage.scss [hunk 2]
```diff
@@ -56,6 +57,7 @@ $my-text-border-radius: 28px 28px 4px 28px;
 
   &--me {
     align-self: flex-end;
+    align-items: flex-end;
```
Comment (0xdevalias) [inline src/components/atoms/ChatMessage/ChatMessage.scss:60]: As above, how does `align-self` and `align-items` differ here?

**Review by 0xdevalias — COMMENTED** [review]
Retroactive review comments.

File: src/components/atoms/ChatMessage/ChatMessage.scss (hunk 1 above)
Comment (mike-lvov) [reply src/components/atoms/ChatMessage/ChatMessage.scss:9]: They target different items. Align-self targets the actual class. align-items targets its children

Comment (0xdevalias) [reply src/components/atoms/ChatMessage/ChatMessage.scss:9]: Right, that makes sense.

**Pull request merged**